script/unicode/Unicode/Reference.lean

```diff
@@ -0,0 +1,16 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jean-Baptiste Tristan
+-/
+import Unicode.Unicode
+
+def referenceTable (ucd : Array UnicodeData) (property : UnicodeData → Bool) : Array Nat :=
+  (ucd.filter property).map fun ucdc => ucdc.codepoint
+
+def referenceSearch (table : Array Nat) (c : Char) : Bool :=
+  table.contains c.toNat
+
+def reference (ucd : Array UnicodeData) (property : UnicodeData → Bool) (c : Char) : Bool :=
```

Comment:
I think this is unused? It's probably best to delete it because calling it is probably never a good idea given that it generated the entire table from scratch each time.